src/components/video-card/video-card.css

```diff
@@ -0,0 +1,8 @@
+#video-card{
+    width: 19rem;
+}
+
+
+#watch-later-btn{
+    font-size: 1.2rem;
```

Comment:
its good to use font-size in em's unit rather than rem's

Reply:
will do in final version